packages/wdio-static-server-service/src/launcher.ts

```diff
@@ -10,14 +10,27 @@ const log = logger('@wdio/static-server-service')
 
 const DEFAULT_LOG_NAME = 'wdio-static-server-service.log'
 
+export interface MiddleWareOption{
+    mount:string;
+    middleware:  string;
+}
+
+export interface FolderOption{
```

Comment:
I believe we can move types to separate file

Reply:
should i move it to "static-server-service.d.ts" file? or should i just create a new file only for types?

Reply:
I did latter in my PRs

Reply:
Changes Done